src/linter.js

```diff
@@ -408,7 +408,11 @@ export default class Linter {
         return this.io.getFiles();
       })
       .then((files) => {
-        return this.scanFiles(Object.keys(files));
+        // Known libraries do not need to be scanned
+        let unknownFiles = Object.keys(files).filter((file) => {
```

Comment:
This is much better, but can we rename this to something a bit more clear? How about `filesWithoutJSLibraries` (or `filesWithoutJSLibs` if it's too long).